website/frontend/src/store/actions/actions.ts

```diff
@@ -244,14 +240,24 @@ export const handleProgramSuccess = () => (dispatch: any, getState: any) => {
   }
 };
 
-export const getNotificationsAction = (params: PaginationModel) => (dispatch: any) => {
-  dispatch(fetchNotificationsAction());
-  notificationService
-    .fetchAllNotifications(params)
-    .then((result: NotificationRPCModel) => {
-      dispatch(fetchNotificationsSuccessAction(result.result));
-    })
-    .catch(() => dispatch(fetchNotificationsErrorAction()));
+export const getNotificationsAction = () => (dispatch: any) => {
+  nodeApi.subscribeProgramEvents((event) => {
```

Comment:
Here you need to check what event is (InitSuccess or InitFailure)

You can use
```javascript
nodeApi.subscribeProgramEvents(({ data, method }) => {
  // method is event name
  // to do stmh
})
```